apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -233,12 +237,13 @@ export function useTrack() {
         eventName,
         {
           ...(analyticsDrawerName ? { drawer: analyticsDrawerName } : {}),
+          ptxEarnEnabled: !!ptxEarnFeatureFlag?.enabled,
```

Comment:
Not sure about this solution.
This method is common across all the app, ptxEarnEnabled  is very specific, I do not think we must include this info in tracks which are not relative to "swap" flow.

Options that I have in mind:
1. Create useSharedStakeTracking method and pass FF value each time that you use it:
`useSharedStakeTracking(ptxEarnEnabled) = {flow: "stake", ptxEarnEnabled }`
`track(...,{..., useSharedStakeTracking(ptxEarnEnabled)}`

2. Create sharedStakeTracking object and keep FF in memory when you call it to the menu:
`sharedStakeTracking = {flow: "stake", ptxEarnEnabled: window.localStorage.getItem("ptxEarnEnabled"); }

Reply:
I went for this solution as it's okay to add extra details that aren't specifically relevant to other flows, eg. tracking adds `hasGenesisPass` to all tracking.  I'll check with Martin to be sure as otherwise we'd end up in a situation where we're trying to add it everywhere for all different scenarios which would be more complex imo than just adding it here

Reply:
Also, checking the usage for the useTrack hook it's only used in a couple of other places besides where I've added it so actually this would suit us well.

Reply:
Which track not relative to `flow: "stake"` do you want to track? The person responsable of track is Amine BELGHAZI

Reply:
Do we must change any "track" to "useTrack"? if not, you wont have the property always.